main.go

```diff
@@ -36,31 +36,28 @@ type VersionReleases map[string]Release
 type ReleasesByChannel map[string]VersionReleases
 
 // fetchGraph fetches the upgrade graph for a given channel and architecture.
-func fetchGraph(client *http.Client, channel, arch string) (*CincinnatiGraph, error) {
-	u, err := url.Parse("https://api.openshift.com/api/upgrades_info/graph")
-	if err != nil {
-		return nil, err
-	}
-	queryParams := u.Query()
+func fetchGraph(client *http.Client, u *url.URL, channel, arch string) (*CincinnatiGraph, error) {
+	modURL := *u
```

Comment:
super nitpick, check for ` nil` - lgtm otherwise

Reply:
good idea.

Reply:
pushed, PTAL